### DISCUSSION TITLE
Does PnP Search support Branded Fonts ?

### DISCUSSION THREAD

**Discussion by Justin-Nash** [microsoft-search/pnp-modern-search#3913] [Q&A]
I have been searching the web, but cant find the answer. Does PnP Modern Search support the new SharePoint Brand Center feature Brand fonts as demonstrated here: https://learn.microsoft.com/en-us/sharepoint/brand-fonts ?

**Comment by kasperbolarsen** [comment]
Based on this issue, https://github.com/microsoft-search/pnp-modern-search/issues/3726 it doesn't look like these new fonts are supported. We would very much welcome a PR to that effect.

**Reply by wobba** [reply]
Here's some resources to help with what is needed to support this https://sharepoint.handsontek.net/2024/04/30/apply-custom-fonts-sharepoint-spfx-solutions/
